server/bin/utils/run-unittest

```diff
@@ -463,7 +463,13 @@ function _setup_state {
         local state
         local user
         while read -r controller name state user; do
-            pbench-state-manager --create=${user} --name=${name} --state=${state}
+            tarball=$(find ${_testdir} -type f -name ${name}.tar.xz)
+            pbench-state-manager --create=${user} --path=${tarball} --state=${state}
+            if [[ ${?} -gt 0 ]]; then
+                echo "ERROR: unable to create ${tarball} dataset" >&2
+                find ${_testdir} -type f -name ${name}.tar.xz >&2
```

Comment:
Repeating the `find` command here seems...unreliable.  Any diagnostics from the `find` command at line 466 have already been sent to `stderr`, and the regular output of that command was captured in `tarball` and reported at line 469; so, at best, the output of line 470 will be redundant, and, at worst, it'll be _different_ from the previous invocation (due to external changes to the environment) which is liable to sow confusion.

(Hopefully, we can replace the `find` command at line 466, in which case there will be no cause to invoke it here....)

Reply:
I'm not concerned about diagnostics; I was trying to figure out why my original attempt didn't work even though I knew a file was there. It was a fairly special case of failure, though, and the second `find` definitely outlived its usefulness. Ah, well.